i18n.js

```diff
@@ -8,6 +8,7 @@ const LANGUAGE_NAMES = {
   en: "English",
   pl: "Polski",
   ru: "Русский",
+  hi: "हिंदी"
```

Comment:
Here you go, you can apply the fix this way. No need to change language code as that's how transifex displays it
```suggestion
  hi_IN: "हिंदी",
```